### PR TITLE
Upgrade nginx-aspnet-mysql to net6.0 🚀

### PR DESCRIPTION
**Pull request by stvansolano**
- Make it as a Minimal API
- Use latest MySqlConnector Nuget package

### CODE REVIEW

**Comment by stvansolano** [comment]
@aiordache @glours please take a look :)

**Comment by stvansolano** [comment]
@glours @aiordache u back? :)

File: nginx-aspnet-mysql/backend/Program.cs [hunk 1]
```diff
@@ -1,84 +1,62 @@
...
+    cmd2.ExecuteNonQuery();
+    
+    for (int i = 0; i < 5; i++)
+    {
+        using MySqlCommand insertCommand = new MySqlCommand( $"INSERT INTO blog (title) VALUES ('Blog post #{i}');", connection, transation);
+        insertCommand.ExecuteNonQuery();
+    }
+    transation.Commit();
+    connection.Close();
 }
```
Comment (glours) [inline nginx-aspnet-mysql/backend/Program.cs:62]: Nit: add new line at the end of the file

File: nginx-aspnet-mysql/backend/aspnetapp.csproj [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="MySqlConnector" Version="0.61.0" />
+    <PackageReference Include="MySqlConnector" Version="2.1.2" />
   </ItemGroup>
 </Project>
```
Comment (glours) [inline nginx-aspnet-mysql/backend/aspnetapp.csproj:10]: Nit: add new line at the end of the file

File: nginx-aspnet-mysql/backend/.gitignore [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+.localhistory/
+
+# BeatPulse healthcheck temp database
+healthchecksdb
+
+# Backup folder for Package Reference Convert tool in Visual Studio 2017
+MigrationBackup/
+
+# Ionide (cross platform F# VS Code tools) working folder
+.ionide/
```
Comment (glours) [inline nginx-aspnet-mysql/backend/.gitignore:350]: Nit: add new line at the end of the file

**Review by glours — APPROVED** [review]
Thanks for you contribution

LGTM

File: nginx-aspnet-mysql/backend/Program.cs (hunk 1 above)
Comment (stvansolano) [reply nginx-aspnet-mysql/backend/Program.cs]: Done.

File: nginx-aspnet-mysql/backend/.gitignore (hunk 3 above)
Comment (stvansolano) [reply nginx-aspnet-mysql/backend/.gitignore]: Done.

**Review by glours — APPROVED** [review]
LGTM

**Pull request merged**